### ISSUE TITLE
etherscan-verify gives me 'metadat.matchAll is not a function'

### ISSUE THREAD

**Issue by web3-guru** [wighawag/buidler-deploy#27]
```
TypeError: metadata.matchAll is not a function
    at extractLicenseFromSources (/Users/web3dotguru/contracts/node_modules/buidler-deploy/src/etherscan.ts:36:60)
    at extractOneLicenseFromSourceFile (/Users/web3dotguru/contracts/node_modules/buidler-deploy/src/etherscan.ts:26:20)
    at submit (/Users/web3dotguru/contracts/node_modules/buidler-deploy/src/etherscan.ts:139:29)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

The reason is, matchAll is only supported on node 12

**Comment by wighawag** [comment]
Hi, thanks for the report. Planning to fix that soon.

**Comment by wighawag** [comment]
fixed in 0.5.13
